docs/pages/gas-manager-admin-api/conditional-gas-sponsorship.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Conditional gas sponsorship
+description: Step-by-step guide to sponsor gas for select transactions and users.
+url: https://docs.alchemy.com/reference/conditional-gas-sponsorship
+slug: reference/conditional-gas-sponsorship
+---
+
+In this guide, you’ll learn how to enable **conditional gas sponsorship** to sponsor gas only for select transactions or users. Ex:
+
+- Sponsor gas only for swaps or txs that interact with your contracts.
+- Sponsor gas only for high-ROI users such as power users of your app or users with proof of humanity.
+
+<Info>
+  **\[Recommended]** Use our [SDK](https://www.alchemy.com/docs/wallets) to
+  create and use wallets. The SDK handles all complexity for you, making
+  development faster and easier.
+</Info>
+
+If you want to use APIs directly, follow these steps.
+
+### 1. Create a Webhook
+
+The webhook decides whether a given transaction should be sponsored. Every time you request gas sponsorship, the Gas Manager will make a POST request to your webhook.
+Your server responds with approval or rejection based on your custom rules.
+
+**Request Payload**
+
+- `userOperation`: The userOp object.
+- `policyId`: The ID of your policy.
+- `chainId`: The ID of the network.
+- `webhookData`: Optional additional data you wish to include in the request, such as proof of humanity.
+
+Here is an example request body:
+
+```json
+{
+  "userOperation": {}, // structure depends on EntryPoint version
+  "policyId": "",
+  "chainId": "",
+  "webhookData": ""
+}
+```
+
+**Expected Result**: Your endpoint should respond with a `200` status code and a JSON body (`{ "approved": true | false }`) indicating whether the userOp passes the custom rules.
+
+### 2. Create a Gas Manager Policy
```

Comment:
Do we need all of this info here? Could we just link to our other doc on gas policies and specifically focus on webhook configuration?